oauth/src/Entity/User.php

```diff
@@ -20,15 +21,24 @@ class User
     #[ORM\Column(type: 'string', length: 50)]
     private $lastName;
 
+    #[ORM\Column(type: 'string', length: 100)]
+    private $email;
+
+    #[ORM\Column(type: 'string', length: 100)]
+    private $password;
+
     public function __construct(
         string $id,
         string $firstName,
         string $lastName,
+        string $email,
     )
     {
         $this->id = $id;
         $this->firstName = $firstName;
         $this->lastName = $lastName;
+        $this->email = $email;
```

Comment:
You have a setter thas is checking if the email is valid, so you can use it here
```
$this->setEmail($email);
```